src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/MSITest.java

```diff
@@ -290,20 +267,19 @@ public void testNumericAkvLocationNoAuth() throws SQLException {
      */
     @Test
     public void testNumericAkvWithBadCred() throws SQLException {

```

Comment:
Can you change the name for this test to be 'testNumericJksWithBadCred'? It's now testing JKS instead of AKV.